Tests/gui/GSCodingFlags/GSCellFlags.m

```diff
@@ -0,0 +1,51 @@
+#include "Testing.h"
+
+#include <Foundation/NSAutoreleasePool.h>
+#include <Foundation/NSValue.h>
+
+#include "GSCodingFlags.h"
+
+int main()
+{
+    CREATE_AUTORELEASE_POOL(arp);
+    GSCellFlagsUnion mask = { { 0 } };
+
+    START_SET("GSCodingFlags GNUstep CellFlags Union")
+    
+    // first make sure flags translate to values
+    mask.flags.state = 1;
+    mask.flags.selectable = 1;
+    mask.flags.scrollable = 1;
+    mask.flags.editable = 1;
+    mask.flags.continuous = 1;
+    mask.flags.useUserKeyEquivalent = 1;
+    mask.flags.truncateLastLine = 1;
+
+#if GS_WORDS_BIGENDIAN == 1
+    pass(mask.value == 0b00010010000000000001110000001001, "mask.flags translates to mask.value");
+#else
+    pass(mask.value == 0b10010000001110000000000001001000, "mask.flags translates to mask.value");
+#endif
+    // reset mask
+    mask.value = 0;
+    mask.flags = (GSCellFlags){0};
+    // now make sure values translate to flags
+#if GS_WORDS_BIGENDIAN == 1
+    mask.value = 0b00000001001000000000110100000001;
+#else
+    mask.value = 0b00000001001000000000110100000001;
+#endif
+
+    pass(mask.flags.state = 1, "state is correctly set");
```

Comment:
You are assigning the value 1 here not testing it. The same is true for the other checks. This explains why you have seven checks but only six bits that are set to one.

Reply:
I see that aplogies. I have been out of a town for a few days but I did just update the test and I ran it locally and it passed, although I do not have access to a big endian system.

Reply:
Now the values are plausible, but the checks are still wrong. Could you please fix that? '=' => '=='